pronto.gemspec

```
@@ -45,7 +45,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('octokit', '~> 4.7', '>= 4.7.0')
   s.add_runtime_dependency('rainbow', '>= 2.2', '< 4.0')
   s.add_runtime_dependency('rexml', '~> 3.2', '>= 3.2.5')
-  s.add_runtime_dependency('rugged', '>= 0.23.0', '<= 1.2.0')
+  s.add_runtime_dependency('rugged', '>= 0.23.0', '< 1.5')
```

```suggestion
  s.add_runtime_dependency('rugged', '>= 0.23.0', '< 2.0')
```
What do you think of relaxing it till 2.0?

---

Makes sense!